.travis.yml

```diff
@@ -54,7 +54,8 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ./CI/travis/before_install_darwin ; fi
 
 script:
-  - ${TRAVIS_BUILD_DIR}/CI/travis/make_linux "$OS_TYPE" "$OS_VERSION"
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ${TRAVIS_BUILD_DIR}/CI/travis/make_linux "$OS_TYPE" "$OS_VERSION" ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ./CI/travis/make_darwin ; fi
```

Comment:
${TRAVIS_BUILD_DIR}/CI/travis/make_darwin
instead of:
./CI/travis/make_darwin
?

Reply:
ack